### DISCUSSION TITLE
Laptop fans (Gigabyte AERO 15)

### DISCUSSION THREAD

**Discussion by wichoLC** [Rem0o/FanControl.Releases#629] [Q&A] [answered]
I have a laptop (AERO 15). When I run the program, I get an error message (NVApi could not initialize).

- Should it work on a laptop too?
- Why is this error popping up?

**Comment by Fr0stX76** [comment]
1. no
2.  see 1

cheers

**Comment by AtA3301** [comment, marked as answer]
> 

Laptops tend to use proprietary controls and even models from the same manufacturer are not the same. Supporting all that is a full project on its own. 

Check out https://github.com/hirschmann/nbfc which has a large collection of laptops included. The software also allows you to create your own laptop entry, but requires knowing how to locate fan control registers manually...

**Reply by Rem0o** [reply]
But..... Took a quick look.... nothing stopping anyone from creating a plugin from the nbfc backend.
